libnopegl/src/pass.c

```diff
@@ -606,13 +598,13 @@ void ngli_pass_uninit(struct pass *s)
     for (size_t i = 0; i < ngli_darray_count(&s->pipeline_descs); i++) {
         struct pipeline_desc *desc = &descs[i];
         ngli_pipeline_compat_freep(&desc->pipeline_compat);
-        ngli_pgcraft_freep(&desc->crafter);
-        ngli_darray_reset(&desc->uniforms_map);
         ngli_darray_reset(&desc->blocks_map);
         ngli_darray_reset(&desc->textures_map);
     }
     ngli_darray_reset(&s->pipeline_descs);
 
+    ngli_darray_reset(&s->uniforms_map);
+    ngli_pgcraft_freep(&s->crafter);
```

Comment:
Shouldn't we drop the uniform after crafter like before (since crafter "depends" on the uniform map)

Reply:
Re-ordered as before. 
I think the crafter does not depend on the uniform map. The uniform map only serves as a list of uniform location + data to know what to update and the associated CPU data.